colibre/scripts/max_temperatures.py

```diff
@@ -33,16 +33,30 @@ def make_single_image(filenames, names, T_bounds, number_of_simulations, output_
     ax.set_ylabel("PDF [-]")
     ax.loglog()
 
+    log_max_Ts = []
+
     for filename, name in zip(filenames, names):
         T_max = get_data(filename)
+        log_T_max = np.log10(T_max)
         h, bin_edges = np.histogram(
-            np.log10(T_max), range=np.log10(T_bounds), bins=250, density=True
+            log_T_max, range=np.log10(T_bounds), bins=250, density=True
         )
         bins = 0.5 * (bin_edges[1:] + bin_edges[:-1])
         bins = 10 ** bins
         ax.plot(bins, h, label=name)
 
-    ax.legend()
+        log_max_Ts.append(f"{name}: {log_T_max.max():3.3f}")
+
+    ax.text(
+        0.05,
+        0.05,
+        "\n".join(["Maximum $\\log_{10} T /$K"] + log_max_Ts),
```

Comment:
We are showing max of T_**max**

```suggestion
        "\n".join(["Maximum $\\log_{10} T_{\\rm max}$ [K]"] + log_max_Ts),
```

Reply:
Logarithmic transforms don't retain units, and can't be applied to quantities with units. That's why we use an algebraic expression for the units (i.e. `log X / Unit` instead of `log X [Unit]`). I will add the `_{\\rm max}` though, thanks!